perf.groovy

```diff
@@ -0,0 +1,76 @@
+// Groovy Script: http://www.groovy-lang.org/syntax.html
+// Jenkins DSL: https://github.com/jenkinsci/job-dsl-plugin/wiki
+
+import jobs.generation.*;
```

Comment:
I think it was so that I could do testing without regenerating the roslyn scripts on the jenkins instance.

@mmitche could you clarify?